### PR TITLE
feat/fix(azure spp): Clean-up docu on Azure SPPs and add missing

### PR DESCRIPTION
**Pull request by SimonTheLeg**
permissions to Replicator SPP

 #CU-6gaafc

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-6gaafc Concept: AAD Tenant Hardening](https://app.clickup.com/t/6gaafc)

File: docs/meshstack.azure.index.md [hunk 1]
```diff
@@ -100,9 +107,127 @@ The Service Principal must be authorized in the scope of the meshcloud AAD Tenan
...
+      }
+    }
+}
+```
+
+### Identity Provider (IdP) Lookup
+
+This principal is only required if the AAD is actually be used as source of user information when assigning users to customers or projects inside the meshPanel. In order to use this functionality, the principal requires the following permissions:
+
+- `User.Read.All`
```
[the remark this answers is not in the document]
Comment (SimonTheLeg) [reply docs/meshstack.azure.index.md:181]: This is what I said two days ago. @sttomm and I found out we don't need Directory.Read.All

**Comment by SimonTheLeg** [comment]
> I approve to get it going, but we should remember that this is now a bit inconsistent to AWS and GCP (there is this Permission Model section as well). So we should update these docs as well in the near future.

But there actually isn't a section called like that 🤔  Or am I going crazy now?
https://docs.meshcloud.io/docs/meshstack.aws.index.html
https://docs.meshcloud.io/docs/meshstack.gcp.index.html

**Comment by tfelix** [comment]
> > I approve to get it going, but we should remember that this is now a bit inconsistent to AWS and GCP (there is this Permission Model section as well). So we should update these docs as well in the near future.
> 
> @tfelix But there actually isn't a section called like that  Or am I going crazy now?
> https://docs.meshcloud.io/docs/meshstack.aws.index.html
> https://docs.meshcloud.io/docs/meshstack.gcp.index.html

This is actually... interesting I thought these sections where there as well. Well okay then. Seems I am crazy though I once had them in place. Then go ahead with merging.

**Pull request merged**